src/lib/server/ApiClient.ts

```diff
@@ -0,0 +1,26 @@
+export type UploadRequest = {
+    ObjectId: number,
+    ObjectKey: string,
+    StatusName: string
+}
+
+export async function getUploadRequest(): Promise<UploadRequest[]> {
+ 
+    try{
+        const uploadRequest: UploadRequest[] =  await fetch('https://gx9mczg2il.execute-api.us-east-1.amazonaws.com/api/v1/users/1/upload/requests')
+                                            .then(function(response) {
+                                                if(!response.ok){
+                                                    throw response.statusText;
+                                                }
+                                                return response.json();
+                                            });
```

Comment:
Please don't mix `await/async` syntax and `Promise` syntax.
We will discuss on this later